Python/Ask_How_To_Qus_using_Voice/how_to.py

```diff
@@ -0,0 +1,48 @@
+import pyttsx3      
+import speech_recognition as sr                     
+from pywikihow import search_wikihow                
+
+engine = pyttsx3.init()
+
+def fun_talk(audio):
+    engine.say(audio)
+    engine.runAndWait()
+
+def get_command():
+
```

Comment:
Add docstring after function declaration

Reply:
yes got it